source/distributed/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@
 INCLUDE_DIRECTORIES(BEFORE ${CMAKE_CURRENT_BINARY_DIR})
 
 SET(_unity_include_src
+  grid_tools.cc
```

Comment:
if you don't need `p::d::Tria`, please put the function inside the already existing `soruce/grid/grid_tools.cc`.